packages/cdktf-cli/lib/models/terraform-cli.ts

```diff
@@ -82,16 +109,26 @@ export class TerraformCli implements Terraform {
       terraformBinaryName,
       ["destroy", "-auto-approve", "-input=false"],
       { cwd: this.workdir, env: process.env },
-      stdout
+      (buffer: Buffer) => {
+        this.onStdOut(buffer);
+        stdout(buffer);
+      },
+      this.onStdErr
```

Comment:
Never noticed this before, but why don't we supply the `-json` option (or whatever it is called) here? Do we even do that somewhere?

Reply:
no seems like we don't